doc/configuration.rst

```diff
@@ -1741,6 +1742,23 @@ flag is passed to ``sphinx-build``. This can be enabled by setting::
     }
 
 
+.. _recommend_examples:
+
+Enabling the example recommender system
+=======================================
+
+Sphinx-Gallery can be configured to generate content-based recommendations for
+the example gallery. This can be enabled by setting ``enable`` to ``True``. The
+number of displayed examples can be changed via the ``n_examples`` parameter,
+e.g.::
+
+    sphinx_gallery_conf = {
+        ...
+        'recommender': {'enable': True, 'n_examples': 5},
+    }
+
+Enabling this feature requires adding `numpy` to the dependencies.
```

Comment:
I think this documentation should be updated to explain what to expect, and give a bit more details on how the recommender system work.